administrative-sdk/pronunciationAnalysis.js

```diff
@@ -481,27 +389,50 @@ class PronunciationAnalysis {
           analysis.score = datum.score;
           analysis.words = PronunciationAnalysis._wordsToModels(datum.words);
         }
-        analyses.push(analysis);
+        return analysis;
       });
-      if (cb) {
-        cb(analyses);
-      }
-    };
+  }
 
+  /**
+   * List all pronunciation analyses in a specific pronunciation challenge.
+   *
+   * @param {PronunciationChallenge} challenge Specify a pronunciation challenge to list speech recordings for.
+   * @param {Boolean} detailed Returns extra analysis metadata when true. false by default.
+   * @param {Sdk~listPronunciationAnalysesCallback} cb The callback that handles the response.
+   * @param {Sdk~listPronunciationAnalysesErrorCallback} [ecb] The callback that handles the error response.
```

Comment:
Missing documentation update. Callbacks are no longer taken and a promise is returned.